SDPi_Supplement/asciidoc/volume1/tf1-ch-10-sdpi-p.adoc

```diff
@@ -370,111 +373,118 @@ This section documents any additional requirements on the profile’s content ac
+The <<vol1_spec_sdpi_p_actor_somds_fhir_gateway>> actor identifies and specifies the logic necessary for connecting a SOMDS network environment with Non-SOMDS Systems that utilize <<ref_hl7_fhir>> for their interoperability protocol.  Generally, this logic is defined in the HL7 <<ref_hl7_fhir_pocd_ig>>.
 
 Gateways implementing this actor can support any of the FHIR architectural approaches:  RESTful, messaging, documents, and SOA.
-For example, a <<actor_somds_fhir_gateway>> can utilize a <<actor_somds_consumer>> to retrieve information from other _<<actor_somds_participant>>_ systems, map it into FHIR Bundle resources and forward it on to non-SOMDS systems in a FHIR message.
+For example, a <<vol1_spec_sdpi_p_actor_somds_fhir_gateway>> can utilize a <<vol1_spec_sdpi_p_actor_somds_consumer>> to retrieve information from other _<<vol1_spec_sdpi_p_actor_somds_participant>>_ systems, map it into FHIR Bundle resources and forward it on to non-SOMDS systems in a FHIR message.
 
-Alternatively, the <<actor_somds_fhir_gateway>> could implement a FHIR server and provide support for systems to discover and retrieve information asynchronously, including the use of FHIR publication / subscription (“pub/sub”) services.
+Alternatively, the <<vol1_spec_sdpi_p_actor_somds_fhir_gateway>> could implement a FHIR server and provide support for systems to discover and retrieve information asynchronously, including the use of FHIR publication / subscription (“pub/sub”) services.
 
-The <<actor_somds_fhir_gateway>> can also support SOMDS services invoked by FHIR-based systems, such as requesting a snapshot of the latest vital signs measurements for a specific patient and triggering a blood-pressure cuff reading.
+The <<vol1_spec_sdpi_p_actor_somds_fhir_gateway>> can also support SOMDS services invoked by FHIR-based systems, such as requesting a snapshot of the latest vital signs measurements for a specific patient and triggering a blood-pressure cuff reading.
 
 [#vol1_clause_sdpi_p_somds_v2_gateway]
 ===== SOMDS V2 Gateway
-
+[#vol1_spec_sdpi_p_actor_somds_v2_gateway, reftext='SOMDS V2 Gateway']
 Actor Summary Definition:
 [none]
-. A <<actor_somds_connector>> that supports use of <<ref_hl7_v2>> for interoperating with Non-SOMDS Systems.
+. A <<vol1_spec_sdpi_p_actor_somds_connector>> that supports use of <<ref_hl7_v2>> for interoperating with Non-SOMDS Systems.
 
-<<actor_somds_v2_gateway>> actors shall be grouped with (inherit from) the abstract _<<actor_somds_connector>>_ actor.
-They shall implement either a <<actor_somds_provider>> and / or <<actor_somds_consumer>> actor.
+<<vol1_spec_sdpi_p_actor_somds_v2_gateway>> actors shall be grouped with (inherit from) the abstract _<<vol1_spec_sdpi_p_actor_somds_connector>>_ actor.
```

Comment:
Throughout the supplement, we're sometimes using Requirement blocks to formulate requirments, and sometimes we end up using inline text (as for this example). We should either refactor to a requirement block or make this a definition style as it was done above:

> Every abstract _<<vol1_spec_sdpi_p_actor_somds_connector>>_ **is grouped** with (inherits from) the abstract _<<vol1_spec_sdpi_p_actor_somds_participant>>_.

Reply:
Alternatively, we could argue that formalized requirements do only occur in TF-2 and TF-3...